### ISSUE TITLE
Build fails with "undefined: SetNumberOfThreads"

### ISSUE THREAD

**Issue by darabos** [lynxkite/lynxkite#403]
The build succeeds in my local repo, but fails in Earthly. The failing step looks like this:

```
+ go build -work -x -o .build/lynxkite-sphynx/lynxkite-sphynx github.com/lynxkite/lynxkite/sphynx/lynxkite-sphynx
WORK=/tmp/go-build4213354748
mkdir -p $WORK/b067/
cat >$WORK/b067/importcfg << 'EOF' # internal
# import config
packagefile os=/opt/conda/go/pkg/linux_amd64/os.a
packagefile strconv=/opt/conda/go/pkg/linux_amd64/strconv.a
packagefile runtime/cgo=/home/mambauser/.cache/go-build/9d/9d4d8aa7e2240c2e2135809074395c90183bddb1ca0fc0a1d6d6899d50aa12da-d
packagefile syscall=/opt/conda/go/pkg/linux_amd64/syscall.a
packagefile sync=/opt/conda/go/pkg/linux_amd64/sync.a
EOF
cd /home/mambauser/lk/sphynx
/opt/conda/go/pkg/tool/linux_amd64/compile -o $WORK/b067/_pkg_.a -trimpath "$WORK/b067=>" -p github.com/lynxkite/lynxkite/sphynx/networkit -lang=go1.18 -complete -buildid fsUdk08KNQ3A5X5Qu-6f/fsUdk08KNQ3A5X5Qu-6f -goversion go1.18.5 -c=4 -nolocalimports -importcfg $WORK/b067/importcfg -pack ./networkit/additions.go
# github.com/lynxkite/lynxkite/sphynx/networkit
networkit/additions.go:17:3: undefined: SetNumberOfThreads
```

It sounds like it could be a version issue since just started happening on `main`. But I've made sure `go`, `swig`, and `libnetworkit` all have the exact same version.

If you look closely, the generated `importcfg` is already wrong. It's almost empty. In the successful build `importcfg` is very long and includes among other things `github.com/lynxkite/lynxkite/sphynx/lynxkite-sphynx` and `github.com/lynxkite/lynxkite/sphynx/networkit`:

```
+ go build -work -x -o .build/lynxkite-sphynx/lynxkite-sphynx github.com/lynxkite/lynxkite/sphynx/lynxkite-sphynx
WORK=/tmp/go-build2728984828
cd /home/darabos/lk
git status --porcelain
cd /home/darabos/lk
git -c log.showsignature=false show -s --format=%H:%ct
mkdir -p $WORK/b001/
cat >$WORK/b001/importcfg.link << 'EOF' # internal
packagefile github.com/lynxkite/lynxkite/sphynx/lynxkite-sphynx=/home/darabos/.cache/go-build/f2/f2ec098b156b46c55600d1d20f07ba929498f3f7e825c6a45f726ac9f1035955-d
packagefile bufio=/home/darabos/miniconda3/envs/lk2/go/pkg/linux_amd64/bufio.a
packagefile bytes=/home/darabos/miniconda3/envs/lk2/go/pkg/linux_amd64/bytes.a
packagefile context=/home/darabos/miniconda3/envs/lk2/go/pkg/linux_amd64/context.a
packagefile encoding/json=/home/darabos/miniconda3/envs/lk2/go/pkg/linux_amd64/encoding/json.a
packagefile fmt=/home/darabos/miniconda3/envs/lk2/go/pkg/linux_amd64/fmt.a
packagefile github.com/apache/arrow/go/arrow=/home/darabos/.cache/go-build/84/8467cc5cb36571adcab7e7e45053f677b548107ba2482a796f558a3d27ef3251-d
...
```

I also see some `git` lines there, so I thought maybe the stripped environment in Earthly is to blame. I cloned the whole repo inside the Earthly environment and it's also failing. I've now created a new local Conda environment and that's failing too. So it's some package version after all, but I'm not sure which one. I'm going through the differences one by one now. 😩

**Comment by darabos** [comment]
I'm not sure what `_go_select` is, but with `_go_select` 2.2.0 we get this error and with `_go_select` 2.3.0 we don't.

**Comment by darabos** [comment]
Fixed in #402.
